replica/database.cc

```diff
@@ -334,6 +334,13 @@ database::database(const db::config& cfg, const utils::directories& dirs, databa
         _cfg.reader_concurrency_semaphore_serialize_limit_multiplier,
         _cfg.reader_concurrency_semaphore_kill_limit_multiplier,
         reader_concurrency_semaphore::register_metrics::yes)
+    , _view_read_concurrency_sem(std::max(1, int(max_count_concurrent_reads * _cfg.materialized_views_rmw_admission_fraction())),
```

Comment:
In the future, I would prefer all the MV-only semaphore code to be in db/view/*.cc, not in database.cc. But we don't need to do it now- we already have a lot of view-specific stuff sprinkled all over the source code, so this wouldn't be the first :-(